_nodes/0011.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Discrete right fibrations
+macrolib: topos
+---
+
+A displayed category $E$ over $B$ is a *discrete right fibration* when for each
+morphism $f:x\to y$ and displayed object $\bar{y}\in E_y$, there exists a unique pair
+of $\bar{x}\in E_x$ and $\bar{f}:\bar{x}\to\Sub{f}\bar{y}$.
+«
+\DiagramSquare{
+  west/style = lies over,
+  east/style = lies over,
+  height = 1.5cm,
+  nw = \bar{x},
+  ne = \bar{y},
+  north = \bar{f},
+  sw = x,
+  se = y,
+  south = f,
+}
+»
+
+TODO: need to require that each $E_x$ is small?
```

Comment:
No need

Reply:
I may be misunderstanding the setup; if we don't require this, how do we obtain from a discrete right fibration a functor `B^op -> Set`?